core/src/lib.rs

```diff
@@ -82,3 +83,8 @@ extern crate solana_metrics;
 #[cfg(test)]
 #[macro_use]
 extern crate matches;
+
+extern crate jemallocator;
+
+#[global_allocator]
+static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
```

Comment:
Aside from the debug capability, is this faster?

Reply:
@sagar-solana i am sure there some benchmark where its faster, but who really knows?

Reply:
So should we be using the std alloc normally and only enable this on some configs?

Reply:
Supposedly it’s much faster, and tunable.  I don’t see a problem with using it by default.